test/Microsoft.Extensions.Configuration.Azure.KeyVault.Test/AzureKeyVaultConfigurationTest.cs

```diff
@@ -106,7 +106,7 @@ public void SupportsReload()
         public void ReplaceDoubleUnderscoreInKeyName()
         {
             var client = new Mock<IKeyVaultClient>(MockBehavior.Strict);
-            var secret1Id = GetSecretId("Section__Secret1");
+            var secret1Id = GetSecretId("Section--Secret1");
```

Comment:
That's why unit tests can pass and the integration is still broken 😛  :trollface: